__fixtures__/unit/helper.js

```diff
@@ -41,9 +41,9 @@ module.exports = {
       log: {
         child: (s) => {
           return {
-            debug: (...s) => console.log(`TEST[debug] > ${JSON.stringify(s)}`),
-            info: (...s) => console.log(`TEST[info] > ${JSON.stringify(s)}`),
-            warn: (...s) => console.warn(`TEST[warn] > ${JSON.stringify(s)}`)
```

Comment:
This was intentionally console.log for debugging purposes while working on the unit tests. Any reason for removing it?

Reply:
there are debugging log every time we ran tests, that's why i replaced it. I just realized we are using logger module, this function is moot. Also, i forgot to use the `Logger` in one file.

Reply:
changed the `checkReRun` file to use `Logger` module